packages/headless/src/controllers/smart-snippet/headless-smart-snippet-interactive-inline-links.ts

```diff
@@ -0,0 +1,142 @@
+import {questionAnswering, search} from '../../app/reducers';
+import {SearchEngine} from '../../app/search-engine/search-engine';
+import {QuestionAnsweringSection} from '../../state/state-sections';
+import {loadReducerError} from '../../utils/errors';
+import {
+  buildInteractiveResultCore,
+  InteractiveResultCore,
+} from '../core/interactive-result/headless-core-interactive-result';
+import {
+  logOpenSmartSnippetInlineLink,
+  logOpenSmartSnippetSuggestionInlineLink,
+} from '../../features/question-answering/question-answering-analytics-actions';
+import {getObjectHash} from '../../utils/utils';
+
+export interface InlineLink {
+  linkText: string;
+  linkURL: string;
+}
+
+/**
+ * @internal
```

Comment:
Why are all of these internal?

Reply:
These are internal because they are initialized by the smart snippet and smart snippet suggestions controllers and therefor there's no reason for developers to initialize them or even reference them.